ocaml/pci/lib_test/test_pci.ml

```diff
@@ -59,23 +59,28 @@ let test_lookup_functions () =
      Device: 82371AB/EB/MB PIIX4 ACPI [7113]
      SVendor:        Red Hat, Inc [1af4]
      SDevice:        Qemu virtual machine [1100] *)
-  let test_lookup = assert_equal ~printer:(fun x -> x) in
+  let test_lookup valid_values value =
```

Comment:
We could have turned this into `test_contains`, but this is more strict :+1:

Reply:
We want to get rid of the pci module very soon anyway, so I just changed the code to pass the tests.

Reply:
A name like `any` or `any_of` would suggest at the call site that we are expecting at least one match.